test/features/caseViewCallbackMessages.feature

```diff
@@ -0,0 +1,90 @@
+@functional @caseViewCallbackMessages
+Feature: Set of scenarios to test callback warnings or errors triggered on CaseView screen
+
+  Background:
+    Given the following definition for 'events'
+      | ID                               | Name                          | Pre condition sate | Post condition state |
+      | createCase                       | Create a case                 |                    | CaseCreated          |
+      | triggerCallbackWarningSingular   | Trigger callback warning      | CaseCreated        | AfterEvent           |
+      | triggerCallbackWarningMultiple   | Trigger callback warnings     | CaseCreated        | AfterEvent           |
+      | triggerCallbackErrorSingular     | Trigger callback error        | CaseCreated        | AfterEvent           |
+      | triggerCallbackErrorMultiple     | Trigger callback errors       | CaseCreated        | AfterEvent           |
+      | triggerCallbackErrorsAndWarnings | Trigger callback errs + warns | CaseCreated        | AfterEvent           |
+    Given the following definition for 'states'
+      | ID          | Name         |
+      | CaseCreated | Case created |
+      | AfterEvent  | After Event  |
+
+    Given I have logged in
+    And a case with CaseView Callback functionality exists
+
```

Comment:
This PR looks very good.
I only have one important comment. According to the testing pyramid principle, we should rely as little as possible on functional tests. Ideally we should strictly test only 'the basics' of the functionality with functional tests, and then rely on faster and less expensive (in terms of maintainability) integration test and unit tests to test all the variations and corner cases of the feature. This also prevents our func tests suite to take a large amount of time to complete.
Because of the above, I would strongly suggest we only leave the following func tests and remove the others:
`Callback errors and warnings displayed when start event triggers multiple errors and warnings`
` Starting event twice will ignore callback warnings generated when start event triggers multiple warnings`